superset/reports/commands/execute.py

```diff
@@ -690,6 +691,7 @@ def validate(  # pylint: disable=arguments-differ
         self, session: Session = None
     ) -> None:
         # Validate/populate model exists
+        logger.info('session is validated: id %s, session: %s', self._model_id, session)
```

Comment:
I would lean more to logging properties of classes. For example, with the session, if there's something in particular that you're looking for, I would log just that rather than the instance. The instance will also have some methods on it that we don't need.